panel/tests/widgets/test_slider.py

```diff
@@ -156,6 +156,68 @@ def test_date_slider(document, comm):
         assert widget.value == 1620777600000
 
 
+@pytest.mark.parametrize("start", [date(2018, 9, 1), datetime(2018, 9, 1)])
```

Comment:
We could also add an option for `np.datetime64`.

Reply:
Just added a new test for np.datetime64.